tools/create_component.py

```diff
@@ -0,0 +1,76 @@
+# create_component.py - Creates React component from sample files
+# into specified folder
+
+# Make python2/3 compatible
+from __future__ import print_function
+import os
+import sys
+import argparse
+from argparse import RawTextHelpFormatter
+
+# Make python2/3 compatible
+if (sys.version_info[0] < 3):
+    input = raw_input # python2 input parses strings as python syntax
+
+PROG_DESC = """
+create_component.py - Creates a React Component with
+given name into the specified folder.
+  - <name>.tsx
+  - <name>.test.tsx
+  - index.tsx
+specify config settings in 'crc.config.json' within
+yarn root director
+"""
+
+parser = argparse.ArgumentParser(description=PROG_DESC,formatter_class=RawTextHelpFormatter)
+parser.add_argument("name",help="Name of React Component")
+parser.add_argument("path",help="Directory path for component, omit filename")
+
+args = parser.parse_args()
+
+# Make required directory folder structure
+component_folder = os.path.join(args.path,args.name);
+if (os.path.isdir(component_folder)):
+    opt = input("Folder exists, confirm potential overwrite(y/n):")
+    if (opt != "y"):
+        print("Script aborted, folder already exists")
+        exit(1)
+else:
+    os.makedirs(component_folder);
+
+# Read template files
+__dir__ = os.getcwd();
+template_component = "template.tsx"  #TODO: Replace with read from crc.config.json
```

Comment:
use all caps for constants 
```suggestion
COMPONENTS_TEMPLATE_PATH = "template.tsx"  #TODO: Replace with read from crc.config.json
```

Reply:
updated